bliss/sleep.py

```diff
@@ -264,7 +267,10 @@ def get_galaxy_loss(self, batch):
         )
 
         recon_losses = -Normal(recon_mean, recon_var.sqrt()).log_prob(images)
-        recon_losses = recon_losses.view(batch_size, -1).sum()
+        recon_losses_old = recon_losses.view(batch_size, -1).sum()
+        # NOTE: why do we need .view here
```

Comment:
Not sure if there's something else subtle going on here but why do we need `.view().sum()` instead of directly `.sum()`?

Reply:
Just using `.sum()` is fine, better even. It's probably the way it is because someone was debugging the code and was looking at just the recon_loss for a particular sample